lib/chef/provider/package/yum/yum_helper.py

```diff
@@ -82,6 +82,28 @@ def any(s):
                 return True
         return False
 
+# Compares a set of candidate packages with the packages installed on a system
+# and returns the set of packages that would not generate a multilib version
+# conflict when installed.
+def remove_multilib_incompat(pkgs):
```

Comment:
remove sounds like packages are gonna be uninstalled. Maybe reject_multilib_incompat ?

Reply:
Agreed.  I like reject better, too.  I'll change this with the next update to this PR.